firtool-resolver/src/main/scala-3/FetchArtifact.scala

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package firtoolresolver
+
+object FetchArtifact {
+  def apply(logger: Logger, defaultVersion: String): Either[String, FirtoolBinary] = {
+    Left("Firtool not found on system: building firtool is not supported in Scala 3")
```

Comment:
This is not right. `llvm-firtool` (the Maven project published by this repo that is just a repackaging of pre-build firtool binaries as Java published artifacts) is a Java project--it has no dependency on the Scala version.

Only `firtool-resolver` needs to be cross-compiled and I am fairly certainly all of this source code can cross compile (or can easily be made to cross-compile).

Reply:
is there a way to get it without coursier? my cursory search indicated that the coursier does not provide a scala3 artifact

Reply:
We can use the Scala 2.13 artifact from Scala 3: https://mill-build.org/mill/fundamentals/library-deps.html#_using_scala_2_13_from_scala_3

Reply:
oh nice, trying..